import-scripts/update-az-mskimpact.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
-export AZ_DATA_HOME=$PORTAL_DATA_HOME/az-msk-impact
-export AZ_MSK_IMPACT_DATA_HOME=$AZ_DATA_HOME/mskimpact
+export AZ_DATA_HOME="$PORTAL_DATA_HOME/az-data"
```

Comment:
Extremely small detail - but would we want to rename the repo to 'az_data' (with an underscore instead of a dash) to match the naming convention for the study?